spring-kafka-test/src/main/java/org/springframework/kafka/test/EmbeddedKafkaBroker.java

```diff
@@ -249,6 +253,18 @@ public void setZkPort(int zkPort) {
 		this.zkPort = zkPort;
 	}
 
+	/**
+	 * Set timeouts for the client to the embedded Zookeeper.
+	 * @param zkConnectionTimeout the connection timeout,
+	 * @param zkSessionTimeout the session timeout.
+	 * @return the {@link EmbeddedKafkaBroker}.
+	 */
+	public EmbeddedKafkaBroker setZkClientTimeouts(int zkConnectionTimeout, int zkSessionTimeout) {
```

Comment:
This is not good.
All the setters has to follow Java Beans convention.
So, please, make separate setters for these options.

Or if we talk about builder API, then it cannot be with a `set` prefix.
Anyway I'd prefer to have these properties to be configured separately.